### PR TITLE
added news for PHPCR ODM 1.3 and updated to PHP 5.6

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lsmith77** [comment]
ping @dbu / @WouterJ

File: src/AppBundle/Resources/data/posts/2015-12-25.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: "symfony-cmf-status-and-phpcr-odm-1-3"
+parent: "/news"
+title: "Symfony CMF status update and PHPCR ODM 1.3 release"
+label: false
+format: markdown
+template: cms/news_detail.html.twig
+publish_start_date: "2015-12-25"
+body: |
+  Progress has not been as stead as we had hoped this year. Behind the scenes we have added several
```
Comment (dbu) [inline src/AppBundle/Resources/data/posts/2015-12-25.yml]: s/stead/steady/

File: src/AppBundle/Resources/data/posts/2015-12-25.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  and 3.0 support. Its not yet entirely clear if we will manage to support 3.0 right away though
+  as it might be more feasible for us to do this only in new major versions which will require 2.8+.
+  However for now we want to maintain compatibility with Symfony 2.3+ if possible until next spring.
+  What is becoming clear however is that PHP 5.3 support is no longer a hard requirement for us. We
+  will simply keep PHP 5.3 compatibility as a best as we can but will switch to PHP 5.4+ if we run
+  into any tricky issues with 5.3.
+
+  That being said, we also wanted to share the news that PHPCR ODM and DoctrinePHPBundle 1.3 have
+  been released. The most important improvements are Symfony 3.0 compatibility while maintaining
+  full compatibility with Symfpny 2.3. Furthermore PHP 7.0 is fully supported. However this
```
Comment (dbu) [inline src/AppBundle/Resources/data/posts/2015-12-25.yml]: s/Symfpny/Symfony/

**Comment by dbu** [comment]
:+1:

did you check whether this runs on platform.sh?

**Comment by dbu** [comment]
ah, its in the PR checks. great, seems to work fine.

**Comment by lsmith77** [comment]
can you tweet out http://cmf.symfony.com/news/symfony-cmf-status-and-phpcr-odm-1-3 ?

**Comment by dbu** [comment]
https://twitter.com/SymfonyCMF/status/681765598150410240

**Pull request merged**